### PR TITLE
fix(aad): once finish update aad should popup message windows

### PR DESCRIPTION
**Pull request by wenytang-ms**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17956371/

### CODE REVIEW

**Review by LongOddCode — DISMISSED** [review]

File: packages/fx-core/src/component/driver/aad/update.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { buildAadManifest } from "./utility/buildAadManifest";
 import { UpdateAadAppOutput } from "./interface/updateAadAppOutput";
 import { InvalidActionInputError, UnhandledError, UnhandledUserError } from "../../../error/common";
 import { updateProgress } from "../middleware/updateProgress";
+import { ViewAadAppHelpLink } from "../../constants";
```
Comment (blackchoey) [inline packages/fx-core/src/component/driver/aad/update.ts:18]: The link is for V4, should we create a new link for V5?

File: packages/fx-core/tests/component/driver/aad/update.test.ts [hunk 2]
```diff
@@ -150,6 +177,83 @@ describe("aadAppUpdate", async () => {
...
+    };
+    const openUrl = sinon.spy(mockedDriverContext.ui, "openUrl");
+    sinon.stub(mockedDriverContext.ui, "showMessage").resolves(ok("Learn more"));
+    const result = await updateAadAppDriver.execute(args, mockedDriverContext);
+    chai.assert.isTrue(openUrl.called);
+    chai.assert.equal(openUrl.getCall(0).args[0], "https://aka.ms/teamsfx-view-aad-app");
+    expect(result.result.isOk()).to.be.true;
+    openUrl.restore();
+  });
+  it("check lerna more link after success without click", async () => {
```
Comment (blackchoey) [inline packages/fx-core/tests/component/driver/aad/update.test.ts:200]: typo, `lerna`->`learn`

File: packages/fx-core/src/component/driver/aad/update.ts (hunk 1 above)
Comment (wenytang-ms) [reply packages/fx-core/src/component/driver/aad/update.ts]: add a new link for v5

File: packages/fx-core/tests/component/driver/aad/update.test.ts (hunk 2 above)
Comment (wenytang-ms) [reply packages/fx-core/tests/component/driver/aad/update.test.ts]: thanks

**Review by blackchoey — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**